### PR TITLE
Adding debug message when passivate method cannot identify entity

### PR DESCRIPTION
**Pull request by sebbes**
Hopefully this will help others to see an unmatched entity when expecting the stopMessage to be sent back to the entity.

### CODE REVIEW

File: akka-cluster-sharding/src/main/scala/akka/cluster/sharding/Shard.scala [hunk 1]
```diff
@@ -227,7 +227,7 @@ private[akka] class Shard(
         messageBuffers = messageBuffers.updated(id, Vector.empty)
         entity ! stopMessage
 
-      case _ ⇒ //ignored
+      case _ ⇒ log.debug("Unknown entity. Not sending stopMessage back to entity.")
```
Comment (johanandren) [inline akka-cluster-sharding/src/main/scala/akka/cluster/sharding/Shard.scala]: Maybe include the path of the `entity` as well?

**Comment by johanandren** [comment]
OK TO TEST

File: akka-cluster-sharding/src/main/scala/akka/cluster/sharding/Shard.scala (hunk 1 above)
Comment (sebbes) [inline akka-cluster-sharding/src/main/scala/akka/cluster/sharding/Shard.scala]: yes, good idea. Will you add this?

Comment (johanandren) [inline akka-cluster-sharding/src/main/scala/akka/cluster/sharding/Shard.scala]: If you cannot find time for it, sure.

Comment (patriknw) [inline akka-cluster-sharding/src/main/scala/akka/cluster/sharding/Shard.scala]: Perhaps even use different log messages for the two cases:
```scala
case Some(_) => log.debug("Passivation already in progress for {}. Not sending stopMessage back to entity.", entity)
case None => log.debug("Unknown entity {}. Not sending stopMessage back to entity.", entity)

```

**Review by patriknw — CHANGES_REQUESTED** [review]
@sebbes Can you please do the suggested changes and push to your branch. Apart from that it is looking good and we can merge.

**Comment by patriknw** [comment]
@sebbes Will you be able to do the final adjustment, or shall I do it for you?

**Comment by sebbes** [comment]
Hi Patrik,

Sorry for the late reply, was on leave. I will adjust it, will do so this
week.

Thanks,
Sebastian

**Comment by ktoso** [comment]
You changed the None case, but a rewording was also proposed for the Some case, see @patriknw's comment :)

Please reword the Some log statement too: `case Some(_) => log.debug("Passivation already in progress for {}. Not sending stopMessage back to entity.", entity)`, after that LGTM

**Review by johanandren — APPROVED** [review]
LGTM

**Review by drewhk — APPROVED** [review]

**Review by patriknw — APPROVED** [review]
LGTM

**Comment by patriknw** [comment]
Thanks @sebbes 
Sorry for long review time.

**Comment by sebbes** [comment]
Hello Patrick,

You're welcome. Thank you for adding this; I am happy to use akka!!!

Regards,
Sebastian Näfe

**Pull request merged**